### PR TITLE
tests: Add codegen to initialize extension functions

### PR DESCRIPTION
**Pull request by artem-lunarg**
This PR implements codegen part.  The intention it to have a separate PR to update all tests and remove function pointer initialization (or to do this incrementally over time). Still two tests were changed to show how it works, one test for instance extension and another one for device extension.

Closes https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/5620

### CODE REVIEW

File: scripts/lvt_file_generator.py [hunk 1]
```diff
@@ -259,9 +261,20 @@ def GenerateFunctionPointerSource(self):
             if item[1] is not None:
                 table += '#endif // %s\n' % item[1]
 
+        table += '\n// Extension function pointers\n'
+        for name in sorted(self.extensionInfo.keys()):
+            table += '// {}\n'.format(name)
```
Comment (ncesario-lunarg) [inline scripts/lvt_file_generator.py:266]: General nit: I personally find the `'...'.format(...)` method much less readable than `f'// {name}'`. `.format` is used throughout the scripts for historical reasons (namely, f-strings were not supported in the past).

**Review by ncesario-lunarg — APPROVED** [review]
Aside from a general python nit, this looks good!

Am I correct in my understanding that these codegen additions are only used in the tests?

File: scripts/lvt_file_generator.py (hunk 1 above)
Comment (artem-lunarg) [reply scripts/lvt_file_generator.py:266]: Yes, it is only for the tests.
I'll check alternative formatting. With python it's like a Groundhog day, I faster forget it than learn.

**Review by spencer-lunarg — APPROVED** [review]
LGTM, all the nasty extensions I know of seem to be handled, excited to start removing more code from the tests!

**Pull request merged**